<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterDialog</class>
 <widget class="QDialog" name="RegisterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>276</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="registerButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pixmap/image/button.png);</string>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pixmap/image/button.png);</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="pwconfirmlabel">
   <property name="geometry">
    <rect>
     <x>54</x>
     <y>150</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>密码确认 :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwConfirmlineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>146</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordlineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>99</y>
     <width>146</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="pwlabel">
   <property name="geometry">
    <rect>
     <x>54</x>
     <y>99</y>
     <width>41</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>密码 :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="userNamelineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>146</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="userNamelabel">
   <property name="geometry">
    <rect>
     <x>54</x>
     <y>50</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>用户名 :</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>349</width>
     <height>276</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(255, 255, 255, 255), stop:0.0146341 rgba(24, 205, 255, 255), stop:1 rgba(255, 255, 255, 0));
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="waitinfo">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pixmap/image/transparent.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>registerButton</zorder>
  <zorder>closeButton</zorder>
  <zorder>pwconfirmlabel</zorder>
  <zorder>pwConfirmlineEdit</zorder>
  <zorder>passwordlineEdit</zorder>
  <zorder>pwlabel</zorder>
  <zorder>userNamelineEdit</zorder>
  <zorder>userNamelabel</zorder>
  <zorder>waitinfo</zorder>
 </widget>
 <tabstops>
  <tabstop>userNamelineEdit</tabstop>
  <tabstop>passwordlineEdit</tabstop>
  <tabstop>pwConfirmlineEdit</tabstop>
  <tabstop>registerButton</tabstop>
  <tabstop>closeButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
